src/components/loadingScreen/loadingScreen.js

```diff
@@ -52,12 +47,10 @@ LoadingScreen.propTypes = {
   staticThrobberImage: PropTypes.string,
   throbberImage: PropTypes.string,
   loadingText: PropTypes.string,
-  standbyText: PropTypes.string,
-  hideDelay: PropTypes.number
+  standbyText: PropTypes.string
 };
 
 LoadingScreen.defaultProps = {
-  hideDelay: 2500,
```

Comment:
@aidenkeating what is the reason for these changes to the loading screen?
Are these cleanup changes as the hideDelay was never used?

Reply:
@david-martin This was due to a change in the way the loading screen is displayed. In `task.js` it's now the default state of the page and will be overridden with the walkthrough when everything is ready. So it's just taking the logic for when the loading screen appears and disappears out of the LoadingScreen component itself and into the implementation. Happy to revert it though.